cypress/support/constants/constants.js

```diff
@@ -124,9 +126,10 @@ module.exports = {
   EXPECTED_JSON_IN_FIREBOLTCALLS: 'Expected JSON data should be defined in fixtures/fireboltCalls/',
   EXPECTED_JSON_IN_VALIDATION_OBJECTS:
     'Expected JSON data should be defined in fixtures/objects/validationObjects/',
-  EXTERNAL_DEVICES_PATH: 'cypress/fixtures/external/devices/',
+  EXPECTING_ERROR: 'expectingError',
+  EXTERNAL_DEVICES_PATH: './node_modules/configModule/cypress/fixtures/devices/',
   EXTERNAL_MODULEREQID_PATH: 'cypress/fixtures/external/objects/moduleReqId/moduleReqId.json',
```

Comment:
@Abhishk123  just for confirmation. did we miss changing to external for below 
EXTERNAL_DEVICES_PATH: './node_modules/configModule/cypress/fixtures/devices/',

Reply:
yes, corrected the path